gossipd/queries.c

```diff
@@ -957,6 +955,45 @@ void maybe_send_query_responses(struct peer *peer)
 	}
 }
 
+bool query_channel_range(struct daemon *daemon,
+			 struct peer *peer,
```

Comment:
weird indent ?

Reply:
GH doesn't like tabs it seems...

Reply:
It has to do with the `+ ` before the indent.  If you print the `diff` out in your terminal, you will see this phenomenon too, not just on Github.  You can ignore it.

This has to do with the fact that `tab` is not 8 spaces, rather it aligns you to a column such that `((column - 1) % 8) == 0`.  Any characters before a tab will not move the position of where the tab terminates.

Reply:
Fwiw the trick I use is to indent with tabs to the precedent line indentation level, then use space to align : git diff, github diff, and my editor are happy.